io.github.liberodark.OpenDrive.json

```diff
@@ -0,0 +1,60 @@
+{
+  "app-id": "io.github.liberodark.OpenDrive",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "18.08",
+  "command": "odrive",
+  "rename-desktop-file": "odrive.desktop",
+  "rename-icon": "odrive",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "separate-locales": false,
+  "finish-args": [
+    "--device=all",
```

Comment:
Now I realize that this might not be needed :thinking:

Reply:
Yes, sounds unlikely.